<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BAAmainMenuWindow</class>
 <widget class="QMainWindow" name="BAAmainMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boarding Actions App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>Welcome to BA list builder v 0.0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="open_list_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>220</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Existing List</string>
    </property>
   </widget>
   <widget class="QToolButton" name="close_app_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>220</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Close App</string>
    </property>
   </widget>
   <widget class="QToolButton" name="new_list_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>220</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>New List</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_menu"/>
    <addaction name="actionOpen_menu"/>
    <addaction name="actionSave_menu"/>
    <addaction name="actionSave_As_menu"/>
    <addaction name="actionExit_menu"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_List">
   <property name="text">
    <string>New List</string>
   </property>
  </action>
  <action name="actionOpen_List">
   <property name="text">
    <string>Open List</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionNew_menu">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_menu">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open existing list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_menu">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As_menu">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit_menu">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
